create-keystone-app/starter/schema.ts

```diff
@@ -1,94 +1,78 @@
+    //   if you want to prevent random people on the internet from accessing your data,
+    //   you can find out more at https://keystonejs.com/docs/guides/auth-and-access-control
+    access: allowAll,
+
+    // this is the fields for our User list
     fields: {
+      // by adding isRequired, we enforce that every User should have a name
+      //   if no name is provided, an error will be displayed
       name: text({ validation: { isRequired: true } }),
+
       email: text({
         validation: { isRequired: true },
+        // by adding isIndexed: 'unique', we're saying that no user can have the same
+        // email as another user - this may or may not be a good idea for your project
         isIndexed: 'unique',
-        isFilterable: true,
       }),
-      // The password field takes care of hiding details and hashing values
+
       password: password({ validation: { isRequired: true } }),
-      // Relationships allow us to reference other lists. In this case,
-      // we want a user to have many posts, and we are saying that the user
-      // should be referencable by the 'author' field of posts.
-      // Make sure you read the docs to understand how they work: https://keystonejs.com/docs/guides/relationships#understanding-relationships
+
+      // we can use this field to see what Posts this User has authored
+      //   more on that in the Post list below
       posts: relationship({ ref: 'Post.author', many: true }),
-    },
-    // Here we can configure the Admin UI. We want to show a user's name and posts in the Admin UI
-    ui: {
-      listView: {
-        initialColumns: ['name', 'posts'],
-      },
+
+      createdAt: timestamp({
+        // this sets the timestamp to Date.now() when the user is first created
+        defaultValue: { kind: 'now' }
+      }),
     },
   }),
-  // Our second list is the Posts list. We've got a few more fields here
-  // so we have all the info we need for displaying posts.
+
   Post: list({
+    // WARNING
+    //   for this starter project, anyone can create, query, update and delete anything
+    //   if you want to prevent random people on the internet from accessing your data,
+    //   you can find out more at https://keystonejs.com/docs/guides/auth-and-access-control
+    access: allowAll,
+
+    // this is the fields for our Post list
     fields: {
-      title: text(),
-      // Having the status here will make it easy for us to choose whether to display
-      // posts on a live site.
-      status: select({
-        options: [
-          { label: 'Published', value: 'published' },
-          { label: 'Draft', value: 'draft' },
-        ],
-        // We want to make sure new posts start off as a draft when they are created
-        defaultValue: 'draft',
-        // fields also have the ability to configure their appearance in the Admin UI
-        ui: {
-          displayMode: 'segmented-control',
-        },
-      }),
```

Comment:
I think we can keep this simpler than a status, if we introduce a `status`, we should do something useful with it.
On that point, we should update the [blog example](https://github.com/keystonejs/keystone/blob/main/examples/blog/schema.ts) to do something with the `status` (like hiding posts that aren't published).